include/filehandler/ifilehandler.h

```diff
@@ -10,15 +10,15 @@ class IFileHandler
 public:
     explicit IFileHandler(std::string const &path);
 
-    virtual bool isFileValid() const;
+    virtual bool isFileValid() const = 0;
```

Comment:
Nice! I did not know about [this](https://stackoverflow.com/a/39388910/1475331). Though, there's no _base_ implementation, right (I don't see any)?

Reply:
In this case, there is no base implementation, indeed. However I have some doubts about making it pure virtual, will see in the future if it's going to stay the same.